docs/administration.emergency-users.md

```diff
@@ -50,44 +50,44 @@ When emergency access is no longer required the following steps will revert perf
 - Remove user from meshProject by opening the project again and navigating to **Project Access** > **Current Access** (performed automatically if expiration date was set)
 - Remove user from meshWorkspace via the [workspace control plane](./meshcloud.workspace.md#managing-your-meshworkspace): go to **Workspace Access** > **Current Access**.
 
-## Access with meshPartner
+## Access with Organization Admin
 
-Even when no workspace manager is available, users with partner admin/employee access can manage permissions for their managed meshWorkspaces.
+Even when no workspace manager is available, users with organization admin/user access can manage permissions for their managed meshWorkspaces.
 
-### Adding emergency user as meshPartner
+### Adding emergency user as Organization Admin
```

Comment:
Lets remove this whole chapter about emergency users same for the remove emergency user